en/starter/hello-world.md

```diff
@@ -19,7 +19,7 @@ const app = express()
 
 app.get('/', (req, res) => res.send('Hello World!'))
 
-app.listen(3000, () => console.log('Example app listening on port 3000!'))
+app.listen(port, host, () => console.log('Example app listening on port 3000!'))
```

Comment:
There is no host nor port variable in the code, which would render this inoperable.